docs/docs/usage.md

```diff
@@ -181,3 +179,20 @@ Triggers the build process of the specified module according to the implicit or
 | `-p (--platform)`   | Optional  |  The name of the target deployment platform. Used only with the `-g (--mtad-gen)` parameter. <br>The supported deployment platforms are: <ul><li>`cf` for SAP Cloud Platform, Cloud Foundry environment  <li>`neo` for the SAP Cloud Platform, Neo environment <li>`xsa` for the SAP HANA XS advanced model</ul> If this parameter is not provided, the `mtad.yaml` file is generated for the SAP Cloud Platform, Cloud Foundry environment.                             | `mbt module-build -m=my_module1,my_module2 -g -p=neo`
 
+
+&nbsp;
+
+### How to generate SBOM file from the project source (BETA)
+
+<b>`mbt sbom-gen`</b>
+
+Generate SBOMs for project modules and merge into one, according to configurations in the MTA development descriptor (mta.yaml)
+
+<b>Usage:</b> `mbt sbom-gen <flags>`
+
+<b>Flags:</b>
+
+| Flag        | Mandatory&nbsp;/<br>Optional        | Description&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;                 | Examples&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;                                    
+| -----------  | -------       |  ----------                          |  -----------------------------
+| `-s (--source)`   | Optional  | The path to the MTA project; the current path is set as the default.                              | `mbt sbom-gen -s C:/TestProject -b sbom-file-gen/test.sbom.xml`
+| `-b (--sbom-file-path)`   | Optional  | The path of SBOM file, the last part of path is file name. <br><ul><li>sbom-file-path can be null, if sbom-file-path is null, default value is <MTA_project_path>/<MTA_project_id>.bom.xml"<li>sbom-file-path can be relative or abs; if relative, it is relative path to project root<li>sbom-file-path's last part is file name <li>only xml file format is supported at present; it means if file suffix is .xml, or no file suffix, xml format sbom will be generated</ul> | `mbt sbom-gen --sbom-file-path sbom-gen/test.sbom.xml`
```

Comment:
The path of the SBOM file. The last part of the path is the file name. <br><ul><li>The sbom-file-path can be null. If the sbom-file-path is null, the default value is the <MTA_project_path>/<MTA_project_id>.bom.xml".<li>The sbom-file-path can be relative or abs; If the path relative, it is the relative path to the project root.<li>The sbom-file-path's last part is the file name. <li>Only the XML file format is currently supported; So, if the file suffix is .xml, or if there's no file suffix, an XML format SBOM will be generated.</ul> | `mbt sbom-gen --sbom-file-path sbom-gen/test.sbom.xml`